doc/nrf/config_and_build/bootloaders/bootloader_downgrade_protection.rst

```diff
@@ -35,6 +35,12 @@ To enable this feature, set the configuration option :kconfig:option:`CONFIG_MCU
    Enabling :kconfig:option:`SB_CONFIG_MCUBOOT_MODE_OVERWRITE_ONLY` prevents the fallback recovery of application images.
    Consult its Kconfig description and the :doc:`MCUboot Design documentation <mcuboot:design>` for more information on how to use it.
 
+   For nRF53 devices, this mode is generally the most appropriate for MCUboot.
```

Comment:
```suggestion
   Enabling ``SB_CONFIG_MCUBOOT_MODE_OVERWRITE_ONLY`` prevents the fallback recovery of application images.
   Consult its Kconfig description and the :doc:`MCUboot Design documentation <mcuboot:design>` for more information on how to use it.

   For nRF53 devices, this mode is the most appropriate for MCUboot.
```

Also, as per agreement, in line 32 the SB_CONFIG_MCUBOOT_MODE_OVERWRITE_ONLY Kconfig should be marked with backticks, as it won't render. This actually needs to be aligned in all the docs in this PR

Reply:
fixed